examples/make_pipeline_database.py

```diff
@@ -0,0 +1,50 @@
+from graphnet.data.pipeline import InSQLitePipeline
+from graphnet.data.constants import TRUTH, FEATURES
+import torch
+torch.multiprocessing.set_sharing_strategy('file_system')
+
+def get_output_column_names(target):
+    if target in ['azimuth', 'zenith']:
+        output_column_names=[target + '_pred', target + '_kappa'] 
+    if target  in ['track', 'neutrino', 'energy']:
+        output_column_names=[target + '_pred']
+    if target == 'XYZ':
+        output_column_names=['position_x_pred','position_y_pred','position_z_pred']
+    return output_column_names
+
+def build_module_dictionary(targets):
+    module_dict = {}
+    for target in targets:
+        module_dict[target] = {}
+        module_dict[target]['path'] = '/home/iwsatlas1/oersoe/phd/oscillations/models/final/dynedge_oscillation_final_%s.pth'%(target) #dev_lvl7_robustness_muon_neutrino_0000/dynedge_oscillation_%s/dynedge_oscillation_%s.pth'%(target,target)
+        module_dict[target]['output_column_names'] = get_output_column_names(target)
+    return module_dict 
```

Comment:
I personally do not feel that the code becomes more readable by wrapping this in yet another class. I think the idea with CombinedModel is really good - perhaps it could be a special case of a more general 'EnsembleModel' class? It would be quite handy!